.cirrus.star

```diff
@@ -35,4 +35,7 @@ def main(ctx):
     # this configuration runs a single linux_aarch64 + macosx_arm64 run.
     # there's no need to do this during a wheel run as they automatically build
     # and test over a wider range of Pythons.
+    if "[skip cirrus]" in dct["message"]:
```

Comment:
This also needs to check for `[skip ci]`.  (Sorry, I didn't think of this until now.)

Reply:
done